packages/react-ssr/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@cdssnc/gcds-components-react-ssr",
+  "version": "0.22.2-canary.0",
+  "author": "Government of Canada / Gouvernement du Canada",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://github.com/cds-snc/gcds-components/issues"
+  },
+  "homepage": "https://design-system.alpha.canada.ca/",
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/esm/index.js",
+      "require": "./dist/cjs/index.js",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./gcds.css": {
+      "import": "./gcds.css",
+      "require": "./gcds.css"
+    },
+    "./server": {
+      "import": "./dist/esm/lib/server.js",
+      "require": "./dist/cjs/lib/server.js",
+      "types": "./dist/types/lib/server.d.ts"
+    },
+    "./client": {
+      "import": "./dist/esm/lib/client/index.js",
+      "require": "./dist/cjs/lib/client/index.js",
+      "types": "./dist/types/lib/client/index.d.ts"
+    }
+  },
+  "directories": {
+    "lib": "lib"
+  },
+  "files": [
+    "dist",
+    "patches",
+    "gcds.css"
+  ],
+  "scripts": {
+    "prebuild": "del-cli --force ./dist && copyfiles -f ../web/dist/components/*.js ./dist/esm/components && copyfiles -f ../../node_modules/@stencil/core/internal/client/index.js ./dist/esm/lib/stencil && copyfiles -f ../../node_modules/@stencil/core/internal/client/shadow-css.js ./dist/esm/lib/stencil && make-dir ./dist/esm && make-dir ./dist/types",
+    "build": "npm run build:esm && npm run build:types && npm run build:cjs",
+    "build:esm": "tsx ./scripts/build.ts",
+    "build:types": "tsc -p .",
+    "build:cjs": "esbuild dist/esm/*.js dist/esm/**/*.js dist/esm/**/**/*.js --log-level=error --outdir=dist/cjs --format=cjs --loader:.js=jsx",
+    "lint": "eslint ."
+  },
+  "dependencies": {
+    "@cdssnc/gcds-components": "^0.22.0",
```

Comment:
Can you up the web components package dependency version to the latest version? `0.22.2`